lib/components/table.vue

```diff
@@ -132,7 +134,7 @@
                 default: () => {
                     if (this.itemsProvider) {
                         // Deprecate itemsProvider
-                        console.warn('b-table: prop items-provider has been deprecated. Pass a function to items instead');
+                        warn('b-table: prop items-provider has been deprecated. Pass a function to items instead');
```

Comment:
I just tried to copy Vue's helper messages. They sort of namespace themselves `[Vue warn]:`. We could even style them all if we wanted to be clever.